shogun-boot/src/main/resources/db/migration/V1.0.2__Init_admin.sql

```diff
@@ -0,0 +1,60 @@
+--SET search_path TO shogun;
+--
+--CREATE EXTENSION IF NOT EXISTS pgcrypto;
+--
+--INSERT INTO users VALUES (
+--    nextval('hibernate_sequence'),
+--    NOW()::timestamp,
+--    NOW()::timestamp,
+--    null,
+--    null,
+--    'admin',
+--    true,
+--    crypt('shogun', gen_salt('bf')),
+--    'admin'
+--) ON CONFLICT DO NOTHING;
+
+--INSERT INTO userinstancepermissions VALUES (
+--    nextval('hibernate_sequence'),
+--    NOW()::timestamp,
+--    NOW()::timestamp,
+--    (SELECT id FROM users WHERE username = 'admin'),
+--    (SELECT id FROM permissions WHERE name = 'ADMIN'),
+--    (SELECT id FROM users WHERE username = 'admin')
+--) ON CONFLICT DO NOTHING;
+--
+--INSERT INTO userclasspermissions VALUES (
+--    nextval('hibernate_sequence'),
+--    NOW()::timestamp,
+--    NOW()::timestamp,
+--    'de.terrestris.shogun.lib.model.Application',
+--    (SELECT id FROM permissions WHERE name = 'ADMIN'),
+--    (SELECT id FROM users WHERE username = 'admin')
+--) ON CONFLICT DO NOTHING;
+--
+--INSERT INTO userclasspermissions VALUES (
+--    nextval('hibernate_sequence'),
+--    NOW()::timestamp,
+--    NOW()::timestamp,
+--    'de.terrestris.shogun.lib.model.User',
+--    (SELECT id FROM permissions WHERE name = 'ADMIN'),
+--    (SELECT id FROM users WHERE username = 'admin')
+--);
+--
+--INSERT INTO userclasspermissions VALUES (
+--    nextval('hibernate_sequence'),
+--    NOW()::timestamp,
+--    NOW()::timestamp,
+--    'de.terrestris.shogun.lib.model.Identity',
+--    (SELECT id FROM permissions WHERE name = 'ADMIN'),
+--    (SELECT id FROM users WHERE username = 'admin')
+--);
+--
+--INSERT INTO userclasspermissions VALUES (
+--    nextval('hibernate_sequence'),
+--    NOW()::timestamp,
+--    NOW()::timestamp,
+--    'de.terrestris.shogun.lib.model.Group',
+--    (SELECT id FROM permissions WHERE name = 'ADMIN'),
+--    (SELECT id FROM users WHERE username = 'admin')
+--);
```

Comment:
Is there any reason why we keep this file?

Reply:
Nope. I'll remove it.